engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/interop/generic/WriteArrayElementNode.java

```diff
@@ -0,0 +1,44 @@
+package org.enso.interpreter.node.expression.builtin.interop.generic;
+
+import com.oracle.truffle.api.interop.InteropLibrary;
+import com.oracle.truffle.api.interop.InvalidArrayIndexException;
+import com.oracle.truffle.api.interop.UnsupportedMessageException;
+import com.oracle.truffle.api.interop.UnsupportedTypeException;
+import com.oracle.truffle.api.nodes.Node;
+import com.oracle.truffle.api.profiles.BranchProfile;
+import org.enso.interpreter.Constants;
+import org.enso.interpreter.dsl.BuiltinMethod;
+import org.enso.interpreter.runtime.EnsoContext;
+import org.enso.interpreter.runtime.builtin.Builtins;
+import org.enso.interpreter.runtime.error.PanicException;
+
+@BuiltinMethod(
+    type = "Polyglot",
+    name = "write_array_element",
+    description = "Write a value from the array specified by the index.",
```

Comment:
```suggestion
    description = "Write a value to the array specified by the index.",
```

Do I misunderstand something or should this be `to` instead of `from`?